pkg/sql/logictest/testdata/logic_test/computed

```diff
@@ -746,3 +746,19 @@ CREATE TABLE x (
 
 query error value type decimal doesn't match type INT8 of column "a"
 INSERT INTO x VALUES(1.4)
+
+# Regression test for #34901: verify that builtins can be used in computed
+# column expressions without a "memory budget exceeded" error while backfilling
+statement ok
+CREATE TABLE builtin_table (x STRING)
```

Comment:
we normally add the issue number to the table name. "t34901" would be the preferred name. Use the same name in the the other file too.

Reply:
Done.